apps/mobile/src/app/settings/wallet/configure/[wallet]/[account]/index.tsx

```diff
@@ -79,12 +79,6 @@ function ConfigureAccount({ fingerprint, accountIndex, account }: ConfigureAccou
                   iconTestID={defaultIconTestId(account.icon)}
                   primaryTitle={account.name}
                   caption={wallet.name}
-                  onPress={() => {
```

Comment:
This is removing the ability to jump to that account on the `Home` screen from settings. Is that intended? 

Is there a way we can do that with the new setup? (Go to Home of a selected account)? 


https://github.com/user-attachments/assets/e9290832-f701-49d4-9135-2a3283a23695